test/test.js

```diff
@@ -53,9 +53,51 @@ test('custom list of lowercase words', async () => {
 test('custom list of excluded words', async () => {
   const result1 = await remark()
     .use(remarkLintHeadingCapitalization, {
-      exclude: ['remark-lint', 'remark']
+      exclude: ['`remark-lint`', '`remark`']
```

Comment:
**issue (if-minor):** @cryspeen Your code changes the way how the strings are processed. Before your change, the `\`\`` were omitted.